docs/tutorials/FlowTutorial.md

```diff
@@ -47,45 +47,79 @@ pipeline.
 
 Use the `bash` shell to run commands and scripts.
 
-Install OpenROAD Flow Scripts. Refer to the [Getting Started with
-OpenROAD Flow](https://openroad-flow-scripts.readthedocs.io/en/latest/user/GettingStarted.html) documentation.
+#### ORFS Installation
 
-If ORFS is already installed but needs updating, run the
-following commands:
+To install OpenROAD-flow-scripts,  refer to the 
+[Build or installing ORFS  Dependencies](https://openroad-flow-scripts.readthedocs.io/en/latest/#build-or-installing-orfs-dependencies)
+documentation.
+
+In general, we recommend using `Docker` for an efficient user
+experience. Install ORFS using a docker as described here
+[Build from sources using Docker](../user/BuildWithDocker.md).
+
+> **Note:**
+> If you need to update an existing docker based installation, then
+> run the following commands:
+> cd OpenROAD-flow-scripts
+> git checkout master
+> git pull
+> ./build_openroad.sh
+
+ORFS installation is complete.
+
+#### Running ORFS inside the Docker
+
+Launch the docker with the `OpenROAD-flow-scripts` container as
+follows:
 
 ```
-cd OpenROAD-flow-scripts
-git checkout master
-git pull
-git submodule update
+export OS_NAME=centos7
+docker run --rm -it -u $(id -u ${USER}):$(id -g ${USER}) -v $(pwd)/flow:/OpenROAD-flow-scripts/flow openroad/flow-$OS_NAME-builder
 ```
+> **Optional:** To launch OpenROAD gui inside the docker, use command
+> from [here](https://openroad-flow-scripts.readthedocs.io/en/latest/user/BuildWithDocker.html#enable-gui-support).
 
-#### Building OpenROAD Locally
+Once you are entered into ORFS container run:
```

Comment:
A couple of lines above, you called the OpenROAD-flow-scripts container. Let's pick one form and keep it consistent across the doc.